doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14192,6 +14582,25 @@ Alternate Fuel types (e.g., FuelType = NaturalGas):
 
 
 
+For the pysics based VRF model (VRF-FluidTCtrl) only:
+
+* VRF Heat Pump Indoor Unit Evaporating Temperature at Cooling Mode [C]
+
+* VRF Heat Pump Outdoor Unit Condensing Temperature at Cooling Mode [C]
+
+* VRF Heat Pump Indoor Unit Condensing Temperature at Heating Mode [C]
+
+* VRF Heat Pump Outdoor Unit Evaporating Temperature at Heating Mode [C]
+
```

Comment:
We will take a deeper look at this when we do the VRF-HR model in the following months. Maybe we can develop a new defrost model then. Thanks for the suggestion!